src/Management/test/Endpoint.Test/Actuators/HttpExchanges/EndpointMiddlewareTest.cs

```diff
@@ -0,0 +1,54 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the Apache 2.0 License.
+// See the LICENSE file in the project root for more information.
+
+using System.Net;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.TestHost;
+using Microsoft.Extensions.Configuration;
+using Steeltoe.Common.TestResources;
+using Steeltoe.Management.Endpoint.Actuators.HttpExchanges;
+using Steeltoe.Management.Endpoint.Configuration;
+
+namespace Steeltoe.Management.Endpoint.Test.Actuators.HttpExchanges;
+
+public sealed class EndpointMiddlewareTest : BaseTest
+{
+    private static readonly Dictionary<string, string?> AppSettings = new()
+    {
+        ["management:endpoints:enabled"] = "true",
+        ["management:endpoints:httpExchanges:enabled"] = "true",
+        ["management:endpoints:actuator:exposure:include:0"] = "httpexchanges"
+    };
+
+    [Fact]
+    public async Task HttpExchangesActuator_ReturnsExpectedData()
+    {
+        WebApplicationBuilder builder = TestWebApplicationBuilderFactory.Create();
+        builder.Configuration.AddInMemoryCollection(AppSettings);
+        builder.AddHttpExchangesActuator();
+        await using WebApplication host = builder.Build();
+        host.UseRouting();
+        await host.StartAsync();
+        HttpClient httpClient = host.GetTestClient();
+        HttpResponseMessage response = await httpClient.GetAsync(new Uri("http://localhost/actuator/httpexchanges"));
+        response.StatusCode.Should().Be(HttpStatusCode.OK);
+        string json = await response.Content.ReadAsStringAsync();
+        json.Should().Be("{\"traces\":[]}");
```

Comment:
Added some line breaks for clarity.
```suggestion
        WebApplicationBuilder builder = TestWebApplicationBuilderFactory.Create();
        builder.Configuration.AddInMemoryCollection(AppSettings);
        builder.AddHttpExchangesActuator();
        await using WebApplication host = builder.Build();

        host.UseRouting();
        await host.StartAsync();
        HttpClient httpClient = host.GetTestClient();

        HttpResponseMessage response = await httpClient.GetAsync(new Uri("http://localhost/actuator/httpexchanges"));
        response.StatusCode.Should().Be(HttpStatusCode.OK);

        string json = await response.Content.ReadAsStringAsync();
        json.Should().Be("{\"traces\":[]}");
```